internal/projects/bgpsessions.go

```diff
@@ -41,19 +41,17 @@ func (c *Client) BGPSessions() *cobra.Command {
   metal project bgp-sessions --project-id 50693ba9-e4e4-4d8a-9eb2-4840b11e9375`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			listOpt := c.Servicer.ListOptions(nil, nil)
-			getOpts := &packngo.GetOptions{Includes: listOpt.Includes, Excludes: listOpt.Excludes}
-			p, _, err := c.BGPConfigService.Get(projectID, getOpts)
+			p, _, err := c.BGPConfigService.FindProjectBgpSessions(context.Background(), projectID).Execute()
```

Comment:
it looks like this has also lost includes/excludes

Reply:
@vasubabu it looks like the other includes/excludes were restored, but this one is still missing